packages/file_selector/file_selector/example/lib/open_image_page.dart

```diff
@@ -15,6 +15,8 @@ class OpenImagePage extends StatelessWidget {
 
   Future<void> _openImageFile(BuildContext context) async {
     // #docregion SingleOpen
+    // TODO(stuartmorgan): https://github.com/flutter/flutter/issues/111906
```

Comment:
For the entries in `#docregion` sections, let's use `ignore_for_file:` at the top of the file instead to get them out of the README excerpts. That's generally not how we want to do ignores, but this will be very short-lived.

Reply:
You've marked this as resolved, but did the opposite of what I requested. As I said, we don't want these TODOs in the README.